src/cardboard-ui.js

```diff
@@ -137,6 +138,8 @@ CardboardUI.prototype.listen = function(optionsCallback, backCallback) {
  * Builds the UI mesh.
  */
 CardboardUI.prototype.onResize = function() {
+  if (!WebVRConfig.ENABLE_CARDBOARD_UI) return false;
```

Comment:
Cardboard UI should be enabled by default. The flag should be DISABLE_CARDBOARD_UI.